bot/exts/evergreen/trivia_quiz.py

```diff
@@ -111,23 +316,39 @@ async def quiz_game(self, ctx: commands.Context, category: str = None) -> None:
                         done_question.append(question_dict["id"])
                         break
 
-                q = question_dict["question"]
-                answer = question_dict["answer"]
+                if "dynamic_id" not in question_dict:
+                    question = question_dict["question"]
+                    answers = question_dict["answer"].split(", ")
+                else:
+                    format_func = DYNAMIC_QUESTIONS_FORMAT_FUNCS[question_dict["dynamic_id"]]
+
+                    question, answers = format_func(
+                        question_dict["question"],
+                        question_dict["answer"],
+                    )
+
+                    answers = [answers]
+
+                embed = discord.Embed(
+                    colour=discord.Colour.gold(),
```

Comment:
I forgot to mention last time too, this should also probably be a constant from Lance's Colours

Reply:
unfortunately it is not... we should add it!

Reply:
done, resolved :D